src/compiler/scala/tools/nsc/reporters/Reporter.scala

```diff
@@ -13,22 +13,22 @@
 package scala.tools.nsc
 package reporters
 
-import scala.reflect.internal.{ForwardingReporter, Reporter => InternalReporter}
-import scala.reflect.internal.util.{Position, ScalaClassLoader}
 import scala.annotation.unused
+import scala.collection.mutable
+import scala.reflect.internal, util.Position, util.ScalaClassLoader
 
 /** This class exists for sbt compatibility. Global.reporter holds a FilteringReporter.
  *  The only Reporter that is *not* a FilteringReporter is the one created by sbt.
  *  The Global.reporter_= setter wraps that in a delegating [[MakeFilteringForwardingReporter]].
  */
-abstract class Reporter extends InternalReporter {
+abstract class Reporter extends internal.Reporter {
```

Comment:
I wound up preferring this expression, like `mutable.Map`. It avoids indirection.

Reply:
It also works better in IntelliJ https://youtrack.jetbrains.com/issue/SCL-16096

Reply:
Ha. Maybe I'll try a contribution over there. After all, that's how Jason got his start.